### PR TITLE
Add Connect Sample Project

### PR DESCRIPTION
**Pull request by caiodias**
This project is meant to show the use of `customerId` and `physicalDeviceIndentifier` parameters of Concierge.connect API.

### CODE REVIEW

File: Concierge/Expose-Bank/Connect-Sample/ViewController.swift [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+
+        self.connectButton.isHidden = Concierge.isConnected
+        self.disconnectButton.isHidden = !Concierge.isConnected
+    }
+
+    @IBAction func connectOnTouchUpInside(sender: UIButton) {
+        let physicalIdentifier = self.physicalDeviceIdentifierTextField.text
```
Comment (mhollis1980FB) [inline Concierge/Expose-Bank/Connect-Sample/ViewController.swift:46]: @caiodias Is there a way to save the values so you can use the check status function?

Comment (caiodias) [reply Concierge/Expose-Bank/Connect-Sample/ViewController.swift:46]: Which values?
the connect and disconnect status?

Comment (mhollis1980FB) [reply Concierge/Expose-Bank/Connect-Sample/ViewController.swift:46]: @caiodias we need some comments to stress the importance of using the same values on subsequent calls to connect

File: Concierge/Expose-Bank/Connect-Sample/ViewController.swift [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+            return
+        }
+
+        vc.customerIdentifier = customerIdentifier
+    }
+
+    @IBAction func connectOnTouchUpInside(sender: UIButton) {
+        let physicalIdentifier = self.physicalDeviceIdentifierTextField.text
+        let customIdentifier = self.customIdentifierTextField.text
+
```
Comment (mhollis1980FB) [inline Concierge/Expose-Bank/Connect-Sample/ViewController.swift:49]:
```suggestion
// The physicalDeviceId provides Flybits' rule engine to include / exclude a user based on their device's current context. By using a custom value, the responsibility of providing a ubiquitous experience between connects and app installs will reside with the app developer. 
```

**Review by mhollis1980FB — APPROVED** [review]

**Pull request merged**